src/protocol/vm/state.rs

```diff
@@ -209,7 +216,8 @@ impl VMPoolState<PreCachedDB> {
                     (Some(code.clone()), code.hash_slow())
                 };
                 engine.state.init_account(
-                    address.parse().unwrap(),
+                    rAddress::parse_checksummed(address, None)
```

Comment:
Is this needed? Also have you tested it? AFAIK addresses are given full lower cap by Tycho, so I'm afraid this will complain about addresses not being checksummed